### PR TITLE
Publish v3.3.0. [release]

### PR DESCRIPTION
**Pull request by ryanbourdais**
Publish v3.3.0. [release]

### CODE REVIEW

File: 3.3/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,97 @@
+# vim:set ft=dockerfile:
+
+# Do not edit individual Dockerfiles manually. Instead, please make changes to the Dockerfile.template, which will be used by the build script to generate Dockerfiles.
+
+# By policy, the base image tag should be a quarterly tag unless there's a
+# specific reason to use a different one. This means January, April, July, or
+# October.
+
+FROM cimg/base:2023.07
```
Comment (JalexChen) [inline 3.3/Dockerfile:9]: 2024.01

**Review by JalexChen — CHANGES_REQUESTED** [review]

**Review by JalexChen — APPROVED** [review]
LGTM

**Comment by ksylvest** [comment]
@JalexChen @ryanbourdais I'm confuse on the process for PRs into this repo. I opened an identical PR a few days ago that got closed without an explanation:

https://github.com/CircleCI-Public/cimg-ruby/pull/150 (also tracked via https://github.com/CircleCI-Public/cimg-ruby/issues/149)

Are you interested in contributions from non CircleCI employees? If not, it might be be worth adding details in README that it isn't open to contributions.

**Comment by ryanbourdais** [comment]
Hi @ksylvest we usually are interested in contributions from non CircleCI employees, this instance was superseded because of customer drive for the 3.3.0 release. There were some changes that needed to be made in #150 and in the interest of time I opened this PR to get the image out as fast as possible to customers. Typically we won't do this and would rather use a non-employee's PR, this is just a special case driven by the release over the holiday.

**Comment by ksylvest** [comment]
@ryanbourdais Thanks for clarifying. I think unfortunately this published image has a bad flag for jemalloc making it unusable for many projects. 

Do you mind reviewing: https://github.com/CircleCI-Public/cimg-ruby/pull/156 / https://github.com/CircleCI-Public/cimg-ruby/issues

**Pull request merged**